test/python_utils/gitdist_UnitTests.py

```diff
@@ -1507,6 +1641,37 @@ def test_dist_repo_status_all(self):
       os.chdir(testBaseDir)
 
 
+  def test_dist_repo_version_table(self):
+    os.chdir(testBaseDir)
+    try:
+
+      # Create a mock git meta-project
+
+      testDir = createAndMoveIntoTestDir("gitdist_dist_repo_version_table")
+
+      os.mkdir("ExtraRepo1")
+      os.mkdir("ExtraRepo2")
+
+      writeGitMockProgram_dist_repo_version_table()
+
+      cmndOut = GeneralScriptSupport.getCmndOutput(
+        gitdistPath + " --dist-use-git=" + mockGitPath \
+          + " --dist-repos=.,ExtraRepo1,ExtraRepo2 dist-repo-version-table",
+        workingDir=testDir)
+      print(cmndOut.decode("ascii"))
+      cmndOut_expected = \
+        "| Repository     | SHA1    | Commit Date         | Author                 | Summary                                        |\n" \
+        "|:-------------- |:-------:|:------------------- |:---------------------- |:---------------------------------------------- |\n" \
+        "| MockProjectDir | e2dc488 | 2019-10-23 10:16:07 | user@domain.com        | Merge Pull Request #1234 from user/repo/branch |\n" \
+        "| ExtraRepo1     | f671414 | 2019-10-22 11:18:47 | wile.e.coyote@acme.com | Fixed a Bug                                    |\n" \
+        "| ExtraRepo2     | 50bbf3e | 2019-10-17 16:32:15 | someone@somwhere.com   | Did Some Work                                  |\n"
+      print(cmndOut_expected)
+      self.assertEqual(s(cmndOut), s(cmndOut_expected))
```

Comment:
@bartlettroscoe, can you help me with this bit here?  `cmndOut_expected` is
```
| Repository     | SHA1    | Commit Date         | Author                 | Summary                                        |
|:-------------- |:-------:|:------------------- |:---------------------- |:---------------------------------------------- |
| MockProjectDir | e2dc488 | 2019-10-23 10:16:07 | user@domain.com        | Merge Pull Request #1234 from user/repo/branch |
| ExtraRepo1     | f671414 | 2019-10-22 11:18:47 | wile.e.coyote@acme.com | Fixed a Bug                                    |
| ExtraRepo2     | 50bbf3e | 2019-10-17 16:32:15 | someone@somwhere.com   | Did Some Work                                  |
```

but `cmndOut` is
```
| Repository     | SHA1    | Commit Date | Author | Summary |
|:-------------- |:-------:|:----------- |:------ |:------- |
| MockProjectDir | e2dc488 |             |        |         |
| ExtraRepo1     | f671414 |             |        |         |
| ExtraRepo2     | 50bbf3e |             |        |         |
```

so it seems like the second, third, and fourth commands in `writeGitMockProgram_dist_repo_version_table()` aren't getting run.  I imagine you can quickly see where I've gone wrong here.  Haven't had much luck with `pdb` yet.  Thanks.

Reply:
Never mind.  I've figured out what's going on.  Should have a fix soon.